modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -79,8 +78,9 @@
   identifiers with it. This is primarily called automatically for the `to-sql` implementation of the
   `BigQueryIdentifier` record type; see its definition for more details."
   []
-  (when (qp.store/initialized?)
-    (some-> (qp.store/database) :details :dataset-id)))
+  "dummy-dataset-id"
+  #_(when (qp.store/initialized?)
+      (some-> (qp.store/database) :details :dataset-id)))
```

Comment:
can this function be removed?

Reply:
This was supposed to have been removed.  Will do that in the next push.